pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumbgppeeringpolicies.yaml

```diff
@@ -132,6 +132,18 @@ spec:
                               whole second.
                             format: duration
                             type: string
+                          gracefulRestartEnable:
```

Comment:
This is just my personal preference (feel free to ignore), but I would prefer using `gracefulRestartEnabled`, as CRD describes the desired state, not an action